taskcluster/ci/test/kind.yml

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+tasks:
+    taskgraph-definition:
+        worker-type: b-linux
+        worker:
+            docker-image: {in-tree: base}
+            max-run-time: 3600
+        description: "Test the full `mozilla_vpn_taskgraph` to validate the latest changes"
+        treeherder:
+            symbol: test-taskgraph-definition
+            kind: test
+            platform: tests/opt
+            tier: 1
+        run:
+            using: run-task
+            use-caches: true
+            cwd: '{checkout}'
+            command: >-
```

Comment:
@strseb I migrated the Linters' "Test Taskgraph Definition"  job to a task. The [checks were failing](https://github.com/mozilla-mobile/mozilla-vpn-client/actions/runs/3091711907/jobs/5002123487) because of the action's checkout behavior. Running these [checks in Taskcluster](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4341/checks?check_run_id=8456268997) tests the Taskgraph in context.

Because the "Test Taskgraph Definition" job was "required" I reckon`test-taskgraph-definition` should be marked as "required" now.